ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -374,7 +374,7 @@ def test_ipa_healthcheck_fips_enabled(self):
         if (
             parse_version(healthcheck_version) < parse_version("0.17")
             and osinfo.id == 'rhel'
-            and osinfo.version_number == (10,0)
+            and osinfo.version_number >= (10,0)
         ):
             # Patch: https://github.com/freeipa/freeipa-healthcheck/pull/349
             pytest.xfail("Patch is unavailable for RHEL 10.0 and "
```

Comment:
Possibly also change the xfail message?

Reply:
@duzda thank you for the review, updated the xfail message as well.